v2 volume: update documents (#829)
longhorn/website: content/docs/1.6.0/deploy/important-notes/index.md

```

Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.

> ### V2 Volume Supports arm64 Platform
> 
> V2 Volume now includes support for the `arm64` platform since Longhorn v1.6.0. Please refer to the [document](../../spdk/prerequisites/) for more information.

Reviewer: ```suggestion
### V2 Volume Support for ARM64 Platform

As of Longhorn v1.6.0, volumes using the V2 Data Engine support the ARM64 platform. For more information, see [Prerequisites](../../spdk/prerequisites/).
```

> ### Longhorn System Upgrade with V2 Volumes
> 
> At present, Longhorn does not support live upgrades of v2 volumes. Prior to initiating the Longhorn system upgrade process, ensure to detach all v2 volumes.

Reviewer: ```suggestion
### Upgrading Longhorn Systems with V2 Volumes

Longhorn currently does not support live upgrading of V2 volumes. Ensure that all V2 volumes are detached before initiating the upgrade process.
```
